homeassistant/components/rfxtrx/__init__.py

```diff
@@ -492,6 +492,7 @@ def __init__(
             identifiers=get_identifiers_from_device_tuple(device_id),
             model=device.type_string,
             name=f"{device.type_string} {device.id_string}",
+            serial_number=device.id_string,
```

Comment:
Is this a serial number?

Reply:
Its an identification number over radio. Bit no not a manufacturing serial number. Which is why i opposed naming it serial in the arch discussion.

Reply:
If it's not a serial number we shouldn't add it here.

Reply:
This really pisses me off. The integration would never ever be able to provide a manufacturing serial number. So it will never conflict. Stupidity.